### PR TITLE
Add etherscan link to transaction overview

### PR DESCRIPTION
**Pull request by MickdeGraaf**
##### Description

Adds a link to the transactions in the overview so users can inspect their transactions on Etherscan

##### Checklist

- [x] Changes don't break existing behavior
- [ ] Tests coverage hasn't decreased

##### Testing

Go to the portfolio page and click the tx link.

##### Refers/Fixes

#54

### CODE REVIEW

**Comment by MickdeGraaf** [comment]
@dimarconicola Can you style the link?

**Comment by dimarconicola** [comment]
lgtm but I would add the following improvements for a next PR:
- [x] etherscan link on the transaction list needs some styling
- [ ] date format is GMT by default but it would be nice to change it according to location

**Review by dimarconicola — APPROVED** [review]

**Comment by dimarconicola** [comment]
tested and lgtm @dmvt @MickdeGraaf

File: src/components/TransactionsTable.jsx [hunk 1]
```diff
@@ -21,6 +23,7 @@ const transactionUSDValue = ({ direction, daiAmount }) => (
 
 const TransactionsTable = () => {
   const transactions = (myAccount.awpTransactions || []).sort((a, b) => b.timestamp - a.timestamp);
+  const { chainId } = useWeb3React();
```
Comment (dmvt) [inline src/components/TransactionsTable.jsx]: Please standardize around the `eth` store instead. `eth.networkId`. We don't always fully initialize web3React.

**Review by dmvt — CHANGES_REQUESTED** [review]

**Comment by dmvt** [comment]
Also as a reminder, if you think it's ready to go, remove the WIP

**Review by dmvt — APPROVED** [review]

**Pull request merged**